manager/state/raft/membership/cluster.go

```diff
@@ -166,18 +166,14 @@ func (c *Cluster) ValidateConfigurationChange(cc raftpb.ConfChange) error {
 func (c *Cluster) CanRemoveMember(from uint64, id uint64) bool {
 	members := c.Members()
 
-	nmembers := 0
+	nmembers := 1
```

Comment:
Better yet, just use `len(members) + 1` in the one place we use it. Problem solved.